<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtUserWidgetsClassClass</class>
 <widget class="QMainWindow" name="QtUserWidgetsClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtUserWidgetsClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>USER MODE</string>
    </property>
   </widget>
   <widget class="QListWidget" name="adminMoviesListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="watchListWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>movies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addWatchListButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add to watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeWatchListButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>remove from watchlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="watchTrailerButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>watch trailer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
